### DISCUSSION TITLE
organizzazione fisica - 11

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#90] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/80610131-81b3-44ee-9675-daee7be26459)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/587614bb-c620-49f4-8435-152e5a562ed5)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/1bf25835-e821-4bb5-b690-cde2f134e753)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e8c089db-96f2-40d1-9e1a-a53f05dc5360)

**Comment by atyion** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/101365532/daef6002-e11b-44c0-a073-f73b50d2e15d)
